### PR TITLE
APIGOV-17864 - fix yaml parsing of camel case paths

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

**Review by kflan-io — APPROVED** [review]

File: pkg/apic/definitions.go [hunk 1]
```diff
@@ -142,3 +146,40 @@ type PlatformTeam struct {
...
+	// first unmarshall it into a map[string]interface
+	var keyInterface map[string]interface{}
+	err := unmarshal(&keyInterface)
+	if err != nil {
+		return err
+	}
+
+	// now loop the keys and lowercase them all
+	for key, val := range keyInterface {
+		if strings.ToLower(key) == key {
```
Comment (sbolosan) [inline pkg/apic/definitions.go:164]: awesome...

File: pkg/apic/definitions.go [hunk 2]
```diff
@@ -142,3 +146,40 @@ type PlatformTeam struct {
 	Name    string `json:"name"`
 	Default bool   `json:"default"`
 }
+
+type oas2Swagger struct {
+	openapi2.Swagger
+}
+
```
Comment (dfeldick) [inline pkg/apic/definitions.go:153]: We decided previously to not put code into the definitions.go file. These files are being excluded from sonar scans.

**Review by dfeldick — CHANGES_REQUESTED** [review]

File: pkg/apic/definitions.go (hunk 2 above)
Comment (jcollins-axway) [reply pkg/apic/definitions.go:153]: I will move it

**Review by dfeldick — APPROVED** [review]

**Comment by sbolosan** [comment]
looks good!

**Review by sbolosan — APPROVED** [review]

**Comment by jcollins-axway** [comment]
@dfeldick - thanks for keeping me honest on that

**Pull request merged**